plan/physical_plans.go

```diff
@@ -530,6 +530,18 @@ const (
 	CompleteAgg
 )
 
+func (at AggregationType) String() string {
```

Comment:
Add comment for the exported function.
Also, it seems that this function is not used?